src/main/java/no/difi/sdp/client2/SikkerDigitalPostKlient.java

```diff
@@ -113,8 +114,11 @@ public ForretningsKvittering hentKvitteringOgBekreftForrige(KvitteringForespoers
      * </ol>
      */
     public void bekreft(ForretningsKvittering forrigeKvittering) throws SendException {
-        EbmsApplikasjonsKvittering kvittering = forrigeKvittering.applikasjonsKvittering;
-        digipostMessageSenderFacade.bekreft(kvittering);
+        bekreft(forrigeKvittering.kanBekreftesSomBehandletKvittering);
+    }
+
+    public void bekreft(KanBekreftesSomBehandletKvittering forrigeKvittering) throws SendException {
```

Comment:
Vi kan la `ForretningsKvittering` implementere `KanBekreftesSomBehandletKvittering` (og bare proxye de metodene som trengs) sånn at vi slipper to metoder her. Is nice.